colabfold/mmseqs/search.py

```diff
@@ -311,7 +311,8 @@ def main():
             query_seqs_cardinality,
         ) in enumerate(queries_unique):
             for seq in query_sequences:
-                f.write(f"{id}\t{raw_jobname}\t{file_number}\n")
+                raw_jobname_first = raw_jobname.split(" ")[0]
```

Comment:
I think it is better to use `raw_jobname_first = raw_jobname.split()[0]` instead since this should also cover tabs.

Reply:
I revised and commited
```suggestion
                raw_jobname_first = raw_jobname.split()[0]
```